scripts/globals/spells/trust/mumor.lua

```diff
@@ -2,19 +2,78 @@
 -- Trust: Mumor
 -----------------------------------
 require("scripts/globals/trust")
------------------------------------
+-----------------------------------------
 local spellObject = {}
 
+local jobs1 =
+{
+    xi.job.WHM,
+    xi.job.RDM,
+    xi.job.SCH,
+    xi.job.PLD,
+}
+
 spellObject.onMagicCastingCheck = function(caster, target, spell)
-    return xi.trust.canCast(caster, spell, 1015)
+    return xi.trust.canCast(caster, spell)
 end
 
 spellObject.onSpellCast = function(caster, target, spell)
+    -- Add synergy variable to the master.
+    local synergy = caster:getLocalVar("mumorSynergy")
+
+    caster:setLocalVar("mumorSynergy", synergy + 1)
+
     return xi.trust.spawn(caster, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
-    xi.trust.message(mob, xi.trust.message_offset.SPAWN)
+    xi.trust.teamworkMessage(mob, {
+        [xi.magic.spell.UKA_TOTLIHN] = xi.trust.message_offset.TEAMWORK_1,
+        [xi.magic.spell.ULLEGORE   ] = xi.trust.message_offset.TEAMWORK_2,
+    })
+
+    -- Locals for synergy
+    local master  = mob:getMaster()
+    local synergy = master:getLocalVar("mumorSynergy")
+
+    -- Dynamic modifier that checks variable on tick to apply
+    mob:addListener("COMBAT_TICK", "MUMOR_CTICK", function(mobArg)
+        if synergy >= 2 then
+            mob:setMod(xi.mod.SAMBA_DURATION, 10)
+        end
+    end)
+
+    -- Listeners to subtract synergy variable if one of the two are killed or released
+    mob:addListener("DEATH", "MUMOR_DEATH", function(mobArg, killer)
+        master:setLocalVar("mumorSynergy", synergy - 1)
+    end)
+
+    mob:addListener("DESPAWN", "MUMOR_DESPAWN", function(mobArg)
```

Comment:
Currently most of the synergy I see is adding and deleting modifiers, though some do change behaviors. I guess big picture thinking would suggest it would be necessary to adjust for that reason.